commands/roles.go

```diff
@@ -75,15 +79,17 @@ func rolesSystemAdminCmdF(c client.Client, _ *cobra.Command, args []string) erro
 		if !systemAdmin {
 			roles = append(roles, model.SystemAdminRoleId)
 			if _, err := c.UpdateUserRoles(user.Id, strings.Join(roles, " ")); err != nil {
-				printer.PrintError(fmt.Sprintf("can't update roles for user %q: %s", args[i], err))
+				err := fmt.Errorf("can't update roles for user %q: %s", args[i], err)
```

Comment:
Same here

Reply:
And you should rename the error instead of shadowing `err`.

Reply:
Ohh sorry I missed that. Did it now. Thank you.